packages/common/src/authorization/scopeAbilityBuilder.test.ts

```diff
@@ -1365,6 +1365,188 @@ describe('scopeAbilityBuilder', () => {
             });
         });
 
+        describe('virtual view permissions', () => {
```

Comment:
added tests to make sure create and delete are 2 se parate scopes, but manage can do both.